lib/generators/factory_girl/model/model_generator.rb

```diff
@@ -75,6 +74,14 @@ def factory_attributes
         end.join("\n")
       end
 
+      def filename
+        if factory_girl_options[:filename_proc].present?
+          factory_girl_options[:filename_proc].call(table_name)
+        else
+          [table_name, filename_suffix].compact.join('_')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.